packages/@vue/cli-plugin-vuex/generator/template/src/store/index.js

```diff
@@ -4,13 +4,6 @@ import Vuex from 'vuex'
 Vue.use(Vuex)
 
 export default new Vuex.Store({
-  state: {
-
-  },
-  mutations: {
-
-  },
-  actions: {
-
+  modules: {
```

Comment:
`modules` field with no actual modules may be confusing for newbies?

For now, we can stick to the original structure and introduce modules later with the planned snippets generator feature.

Reply:
How about the state, actions and also modules? codemods will work better if the `modules` is present here.

Reply:
Yeah that seems fine.